libs/data-mapper-v2/src/components/addSchema/styles.ts

```diff
@@ -1,7 +1,17 @@
 import { makeStyles, shorthands } from '@fluentui/react-components';
 
 export const useStyles = makeStyles({
-  wrapper: {
+  drawerWrapper: {
+    display: 'flex',
+    flexDirection: 'row',
```

Comment:
NIT: I think the default flex-direction for display flex is row. Maybe stating it twice is not the best practice?